models/repo.go

```diff
@@ -439,7 +439,7 @@ type CloneLink struct {
 	Git   string
 }
 
-func (repo *Repository) cloneLink(isWiki bool) *CloneLink {
+func (repo *Repository) cloneLink(isWiki bool, signedUserName string) *CloneLink {
```

Comment:
Call `userName` is good enough, `signedUserName` is narrowing the actual usage of this method.